docs/Converters/IXItoBIDS.md

```diff
@@ -0,0 +1,52 @@
+# `ixi-to-bids` – Conversion of Information eXtraction from Images (IXI) to BIDS format
+??? quote "Dataset Description"
+    [IXI webpage](https://brain-development.org/ixi-dataset/)
```

Comment:
I think it'd be easier to read if we had the link attached to the name instead of explicitly displaying it at the beginning:

```suggestion
```